docs/7.x/testplan.md

```diff
@@ -311,29 +311,44 @@ license:
     enabled: true
 ```
 
-- [ ] Generate a license with encryption key:
+- [ ] Generate two licenses:
 ```bash
-$ gravity license new --max-nodes=3 --valid-for=24h --ca-cert=domain.crt --ca-key=domain.key --encryption-key=qwe123 > license.pem
+$ gravity license new --max-nodes=3 --valid-for=24h --ca-cert=domain.crt --ca-key=domain.key --encryption-key=qwe123 --state-dir $(mktemp -d) > license.pem
+$ gravity license new --max-nodes=3 --valid-for=48h --ca-cert=domain.crt --ca-key=domain.key --encryption-key=qwe123 --state-dir $(mktemp -d) > license.new.pem
 ```
+ `--state-dir` may not be necessary. Check [#1852](https://github.com/gravitational/gravity/issues/1852)
 
 - [ ] Build an encrypted cluster image:
 ```bash
 $ tele build app.yaml --ca-cert=domain.crt --encryption-key=qwe123
 ```
 
 - [ ] Verify can install in wizard UI mode.
-  - [ ] Verify license prompt appears in the UI.
-  - [ ] Insert the generated license and verify the installation succeeds.
-  - [ ] Verify license can be updated via cluster UI after installation.
+ - [ ] Verify license prompt appears in the UI.
```

Comment:
nit: Add additional space if this was intended to be indented.

Reply:
Fixed in 5770c9e.